apps/minaintyg/src/components/error/ErrorPageHero.tsx

```diff
@@ -7,13 +7,15 @@ import { PageHero } from '../PageHero/PageHero'
 import { PageHeroActions } from '../PageHero/PageHeroActions'
 import { SupportLink } from './SupportLink/SupportLink'
 
-export const ErrorType = z.enum(['unknown', 'login-failed'])
+export const ErrorType = z.enum(['unknown', 'login-failed', 'unavailable'])
 export type ErrorTypeEnum = z.infer<typeof ErrorType>
 const ActionType = z.enum(['start'])
 export type ActionTypeEnum = z.infer<typeof ActionType>
 
 function getErrorHeading(type?: ErrorTypeEnum) {
   switch (type) {
+    case ErrorType.enum.unavailable:
```

Comment:
Should this be case ErrorType.enum['unavailable'] ?

Reply:
Nah, this gets formatted automatically. It's only needed for `login-failed` because of the `-`